ecosystem/typescript/sdk/src/generated/core/request.ts

```diff
@@ -49,7 +49,6 @@ const base64 = (str: string): string => {
     try {
         return btoa(str);
     } catch (err) {
-        // @ts-ignore
         return Buffer.from(str).toString('base64');
```

Comment:
ah, we will get this removed as well in another PR. Otherwise, the transpiled code doesn't run in browser.